packages/visual-editor/src/hooks/useEditorSubscriber.ts

```diff
@@ -220,6 +222,7 @@ export function useEditorSubscriber() {
           }
           // Update the tree when all necessary data is fetched and ready for rendering.
           updateTree(tree);
+          manuallyFireCanvasGeometryUpdate(tree);
```

Comment:
do we need to trigger it _manually_ since it's already subscribed to all dom changes?

Reply:
When I unbound the asset of an image node, it doesn't trigger a DOM update. I don't know why that is, but I thought it might be a good safety mechanism to enforce an update when the tree changed just to be safe. Since we debounce the updates, I don't expect a major performance issue by doing so.

We could spend more time later on why the described example doesn't trigger the update but at least, we now have a stable system to work with :)

Reply:
Interesting, I expected it to work differently. Is it possible that the dom doesn't change and we update the image using css only?

Reply:
maybe call this function `sendCanvasGeometryUpdate` or something more generic?